editor/src/clj/editor/protobuf.clj

```diff
@@ -360,7 +359,7 @@ Macros currently mean no foreseeable performance gain however."
                                      (->CamelCase (FilenameUtils/getBaseName (.getName file))))
                          inner-cls (desc-name desc)]
                      (str package "." outer-cls "$" inner-cls)))]
-    (Class/forName cls-name)))
+    (workspace/load-class! cls-name)))
```

Comment:
We need all classes to use the same class loader.
Otherwise I got weird type mismatch issues. (instances were created using different class loaders)